tobac/tests/test_utils.py

```diff
@@ -0,0 +1,79 @@
+"""
+Tests for the tobac.utils.general module.
+
+"""
+
+
+import numpy as np
+from iris.coords import DimCoord
+from iris.cube import Cube
+from tobac.utils.general import get_spacings
+import datetime
+import cf_units
+import pandas as pd
+import pytest 
+
+
+
+
+# define hourly time range for random cube 
+timerange = pd.date_range(datetime.datetime(2021,1,1), datetime.datetime(2021,12,1), freq= 'h')
```

Comment:
I am actually not sure. Generating test data inside the function also makes sense, but with the parametrizing fixture `@pytest.mark.parametrize()` it is possible to test multiple datasets or sets of arguments in general. So if the test functions are going to be extended, this could be in advantage, because it can directly be tracked if the test fails for a specific combination of arguments.